docs/standard/library-guidance/cross-platform-targeting.md

```diff
@@ -57,7 +57,7 @@ To shield your consumers from having to build for individual frameworks, you sho
 
 > The .NET Standard assembly will automatically be used by NuGet. Targeting individual .NET implementations increases the `*.nupkg` size for no benefit.
 
-**✔️ CONSIDER** adding a target for `net461` when you're offering a `netstandard2.0` target. 
+**✔️ DO** add a target for `net461` when you're offering a `netstandard2.0` target. 
```

Comment:
The language is by-design. It's only a suggestion for library authors that expect to have customers on .NET Framework < 4.7.2. If you don't have .NET Framework customers (because you're only sharing between, say, UWP and Xamarin) or if virtually all customers are on 4.7.2 or higher, than this just adds work with close to zero gain.

Reply:
@terrajobst The problem is that as a library author, I have no idea what framework consumers of my library are going to be using. Given that I have no way to actually enforce that they need to be using `net472` or higher, that means I have no choice but to be defensive here and also ship a `net461` assembly.
 
I don't think the current language on the document is enough to adequately explain the situation.

Though, what I really want to see is some way to author a `netstandard2.0` package and have everything block installation on `net461` - `net471`.

Reply:
> Though, what I really want to see is some way to author a `netstandard2.0` package and have everything block installation on `net461` - `net471`.

You can do this by adding a targets file to your NuGet package. [Here is a demo](https://github.com/terrajobst/require-net472).

Reply:
> You can do this by adding a targets file to your NuGet package. Here is a demo.

That does look like it should work, though I'd ideally like to see that built in to NuGet vs. needing to custom author a target for it.

Reply:
Maybe, but this a point in time issue. Even they would be building it into NuGet, you'd likely require a new NuGet client/VS version which in practice means you'll want the targets file as a fallback.